### PR TITLE
ADA Updates

### PR DESCRIPTION
**Pull request by fditata**

### CODE REVIEW

File: gatsby-browser.js [hunk 1]
```diff
@@ -4,56 +4,210 @@
  * See: https://www.gatsbyjs.com/docs/browser-apis/
  */
 
-// You can delete this file if you're not using it
-
 exports.onInitialClientRender = () => {
```
Comment (nonahayrapetyan) [inline gatsby-browser.js:7]: Fab, why are you doing all of this with pure js and in this file. What about adding those labels and everything that you're doing here in each component directly?

Comment (fditata) [reply gatsby-browser.js:7]: Nona, that was my idea initially. But some labels, inputs, etc. that I have to edit come directly from some library. For example, the price slide on the home page. So I found this solution to be able to apply the changes. Also in this way, I can apply changes more generally and avoid more ADA issues.

Comment (nonahayrapetyan) [reply gatsby-browser.js:7]: Does this generic solution cover every case? I think there should be specific cases for each input,label, etc.
For example you gave "Linked Image" alt to all images but each image should have it's own descriptive alt text.
This is one example but I think it applies to other cases as well (input, etc).

Comment (fditata) [reply gatsby-browser.js:7]: Yes, you're right of course. I just checked the code again to be sure, and for example, in the case of the images, when the image doesn't have an alt text, we set a generic alt text. The other functions I did, work in the same way.

Comment (nonahayrapetyan) [reply gatsby-browser.js:7]: I think it should be individual for each one. Please take the demo url generated in this pr, run ADA reports with ADA tools and see if these changes fixed the bugs. If yes we can merge.

**Review by nonahayrapetyan — APPROVED** [review]

**Pull request merged**